py/desispec/workflow/desi_proc_funcs.py

```diff
@@ -525,18 +532,20 @@ def create_desi_proc_batch_script(night, exp, cameras, jobdesc, queue, runtime=N
 
     return scriptfile
 
-def find_most_recent(night, file_type='psfnight', n_nights=30):
+def find_most_recent(night, file_type='psfnight', exp='00000000', cam='r', n_nights=30):
```

Comment:
Elsewhere we've been standardizing on expid being an integer not a string.  In this case, it appears that this function only works for night-based `file_type` anyway (e.g. psfnight, fiberflatnight, but not psf or fiberflat) since it doesn't actually do anything with `exp` except pass it forward to `findfile`, which crashes if given a str expid for file formats that actually need an expid.

Suggestion: drop `exp` from the arguments and make it clear in the docstring that this searches back by nights, not by expids.  The findfile below could be `nightfile = findfile(file_type, test_night_str, camera=cam)` (i.e. it doesn't need an expid option for filetypes that are night-based).

Reply:
Sounds good. I just noticed that the current master version doesn't work because cam is undefined, so I made this as a quick change. I didn't put much thought into the expid choice. I will remove expid and update the docstring.